overlord/snapstate/autorefresh_gating_test.go

```diff
@@ -149,8 +163,19 @@ slots:
         interface: iface1
 `
 
+const coreYaml2 = `name: core
+type: os
+version: 1
```

Comment:
was the intention to set this to 2 to distinguish it from coreYaml?

Reply:
Yes, I wanted two variants with different interfaces for different test cases.

Reply:
should we have version: 2, here then? same for the other one

Reply:
FWTW version isn't relevant for the tests, it's just to make yaml validation happy.

Reply:
I know I just find it slightly confusing that we have to different core... yaml etc but with the same version?

Reply:
Ok, the only reason was using TestInterface in some tests and real (mir/desktop) in others. I've simplified it now, there is no need for TestInterface since I'm pulling builtin anyway (which I initially hoped could be avoided), so I dropped these two extra definitions for core/snapd.